docs/deploy/11-validation.md

```diff
@@ -201,10 +190,10 @@ If you configured your third-party images to be pulled from your Azure Container
 
    ```kusto
    ContainerRegistryRepositoryEvents
-   | where OperationName == 'Pull'
+   | where OperationName == 'importImage'
    ```
 
-1. You should see logs for kured. You'll see multiple for some as the image was pulled to multiple nodes to satisfy ReplicaSet/DaemonSet placement.
+   > :bulb: you should see the `importImage` operation for the traefik image you imported earlier in the [Configure AKS Ingress controller with Azure Key Vault integration step](./09-secret-management-and-ingress-controller.md).
```

Comment:
Why can't we show the traefik container being pulled?  Having images being pulled has more security and dugging value than what images were pushed to the registry.